src/modules/system/System.cpp

```diff
@@ -104,10 +117,18 @@ bool System::openURL(const std::string &url) const
 	pid_t pid;
 	const char *argv[] = {"xdg-open", url.c_str(), nullptr};
 
+#ifdef LOVE_HAS_POSIX_SPAWN
 	// Note: at the moment this process inherits our file descriptors.
 	// Note: the below const_cast is really ugly as well.
 	if (posix_spawnp(&pid, "xdg-open", nullptr, nullptr, const_cast<char **>(argv), environ) != 0)
 		return false;
+#else
+	pid = fork();
+	if (pid == 0) {
```

Comment:
The `{` should be in its own line, as per https://love2d.org/wiki/Code_Style

Reply:
Done